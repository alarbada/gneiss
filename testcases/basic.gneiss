<g-component name="basic_interpreted">
    <div>
        <h1>Test test</h1>
        <p>test test </p>
    </div>
</g-component>
<g-component name="basic_compiled">
    <div>
        <h1>Test test</h1>
        <p>test test </p>
    </div>
</g-component>
