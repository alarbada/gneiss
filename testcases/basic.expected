    <div>
        <h1>Test test</h1>
        <p>test test </p>
    </div>
